src/include/rocblas_float8.h

```diff
@@ -158,7 +160,7 @@ struct ROCBLAS_EXPORT rocblas_f8
     {
     }
     // constructor from bfloat16
-    explicit HIP_HOST_DEVICE rocblas_f8(rocblas_bfloat16             v,
+    explicit HIP_HOST_DEVICE rocblas_f8(rccl_bfloat16             v,
```

Comment:
why rocblas_bfloat16 won't work?

Reply:
It's strange that this header depends on rccl things.

Reply:
offline discussed, agree to remove the bfloat16 from bf8 header.